Sources/Instructions/Helpers/Public/CoachMarkHelper.swift

```diff
@@ -61,9 +62,10 @@ public class CoachMarkHelper {
         withArrow arrow: Bool = true,
         arrowOrientation: CoachMarkArrowOrientation? = .top,
         hintText: String,
-        nextText: String? = nil
+        nextText: String? = nil,
+        nextLabelPosition: CoachMarkNextLabelPosition? = .none
```

Comment:
It doesn't need to be optional in my opinion, because there's always going to be a position.

```suggestion
        nextLabelPosition: CoachMarkNextLabelPosition = .trailing
```